### PR TITLE
HomePage + Live Search Filter

### PR DESCRIPTION
**Pull request by vuminhhieu1311**
## Related Tickets
- [#40168](https://edu-redmine.sun-asterisk.vn/issues/40168)

## WHAT (optional)
- Design Homepage
- Load jobs to homepage screen (using infinite scroll pagination)
- Add live search filter for jobs
## Evidence (Screenshot or Video)
ezgif-3-d301ffe164bf

### CODE REVIEW

**Review by nvvu99 — APPROVED** [review]
Ok

**Comment by vuminhhieu1311** [comment]
Ready

File: resources/views/layouts/homepage.blade.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    @if (Auth::check())
+        @if (Auth::user()->isEmployee() || Auth::user()->isAdministrator())
+            @include('layouts.employee_header')
+        @elseif (Auth::user()->isEmployer())
+            @include('layouts.employer_header')
+        @endif
+    @else
+        @include('layouts.dark_header')
+    @endif
```
Comment (huyenttm-0901) [inline resources/views/layouts/homepage.blade.php:70]: dùng includeWhen cho ngắn gọn nè em

Comment (vuminhhieu1311) [reply resources/views/layouts/homepage.blade.php:70]: ok ạ

**Comment by vuminhhieu1311** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**